.env

```diff
@@ -1,4 +1,8 @@
-# On running the 'build' commands this file is replaced by the files in `env-config/`
-# To ignore and keep this file we have a command in `postshrinkwrap` which assumes this file is not changing
-# The command is `git update-index --assume-unchanged .env` 
-# Source: https://wildlyinaccurate.com/git-ignore-changes-in-already-tracked-files/
+SIMORGH_BASE_URL=http://localhost.bbc.com:7080
```

Comment:
This file needs resetting, it appears `git update-index --assume-unchanged .env` didnt run on your machine right 🙈 If you do `git checkout latest .env` commit & push it, and then do `git update-index --assume-unchanged .env` it should stop it happening again :)

Reply:
Ah! Whoops 😅 thanks!